### PR TITLE
Adds basic chems to underused plants

### PR DESCRIPTION
**Pull request by WaterStBlues**
[Changelogs]: 

Added basic chems to some underused plants in botany. Chems were mainly chosen for ones missing from some very basic recipes and medicines.
I realize there might be a backlash, but also consider that botany is just as dangerous as it is timesinking. More on this below.
_first real PR pls be gentle_

:cl: Botany Chem Tweaking
tweak: adds sulfur to onions
tweak: add liquidgibs to meatwheat
tweak: add oxygen to space tobacco
tweak: add carbon to tower caps
tweak: add iron/aluminium to steel caps
tweak: add ethanol to moonflowers
tweak: add nitrogen to cabbage
tweak: replace corn oil in sunflowers with regular oil
tweak: add sodium to red beets
/:cl:

[why]: 
Currently it takes less than 30 minutes for a skilled botanist to reliably produce near-infinite amounts of deadly weapons such as throwables containing amantin/coiine/cyanide, tesla shocks and frost oil/lube spam. You don't see this all that often due to how time and labor intensive this process is, and usually when it goes well the results are more impressive and hilarious than frustrating for the victims.

I don't think that adding basic chems and expanding the potential range of botany is something that shouldn't be tried. Don't be scared of organic autism. This PR was also geared towards medicine production. There isn't much that can be accomplished with this PR that isn't currently possible with a chem dispenser and a syringe.

--- 
Individual explanations:

- Sunflower will now produce 10% (regular) oil instead of corn oil, dunno why this wasn't the case to begin with.

- Adding nitrogen to cabbage seemed to make the most sense, alternatively coffee could be used.

- Added oxygen to space tobacco (as it's the EVA plant, just made sense)

- Added aluminum and iron to steel cap logs - I swear this used to be the case with iron; the wiki even still says so. It's nice when Botany can produce EMP 'nades to respond to an AI/mech threat. Regardless, this will still take a while as separated contents are ded. Thermite throwables will also be possible, but will require a lot of effort as the max thermite size will be 20u (as 50u and a significant amount of time are usually required for a wall.)  Aluminum is necessary for styptic powder, organic flashbangs and more. 

- Liquidgibs added to meatwheat (funny applications and could potentially lead to soapmemes) - in retrospect, could instead be added to blood matoes

- Ethanol added to moonflowers. I wanted to put this on something else, but moonflowers are the only boozy plant with a whopping 20% moonshine production. An extra 10% ethanol will make them more potent, but still in the same ballpark.

### CODE REVIEW

**Comment by Shardj** [comment]
Adding oxygen means saltpetre can be made in botany now, as can thermite and other fuckery. I'm sure there are other chems there which shouldn't be too but I don't play botany.

**Comment by Shardj** [comment]
Also the last thing that botany needs is a major buff

**Comment by WaterStBlues** [comment]
> Adding oxygen means saltpetre can be made in botany now, as can thermite and other fuckery. I'm sure there are other chems there which shouldn't be too but I don't play botany.

I don't see the issue with botany making (max 10u, recipe requires 3 parts oxygen) saltpetre plants.
As for thermite, read the post - anyone with a chem dispenser and and some beakers can cause much more damage, much faster. 20u isn't enough to destroy a wall as of the last time I used it.

**Comment by 4dplanner** [comment]
I'd just like to comment on the iron part - iron production was a balance issue when separated chems were in, but is no longer, so that bit should be fine. Botany can already grind to make iron with a few trivial extra steps anyway.

**Comment by Teepeepee** [comment]
yeah it's 10u salt but you can just plant many of them or up their yield, with this you would bypass not only the chemist but most of the mechanics that affect potency since you'll just max it with modified wheat or other fast production plants in the first 10 minutes

**Comment by necromanceranne** [comment]
Botany about to bring about a new fucking dark age. Love it.

**Comment by Shardj** [comment]
@WaterStBlues I don't have to play botany to see how strong it is. I can also see that with oxygen you can make plants that give bidcarine, morphine, godamn stabilizing agent, or fucking space lube.

I don't know about you but I would fucking hate space lube liquid plants being thrown around

**Comment by 4dplanner** [comment]
Morphine is already directly a plant chem

**Comment by 4dplanner** [comment]
So is bicaridine, for that matter

**Comment by WaterStBlues** [comment]
> yeah it's 10u salt but you can just plant many of them or up their yield, with this you would bypass not only the chemist but most of the mechanics that affect potency since you'll just max it with modified wheat or other fast production plants in the first 10 minutes

if you read my [post](https://tgstation13.org/phpBB/viewtopic.php?f=5&t=21337) you'll notice that 10u additions are too low to even brush the prouduction speed threshold, but regardless the current botany meta involves building a chem dispenser from tech storage/stealing it outright anyway. Regardless, with oxygen added to space tobacco, you require mutagen to acquire the gene in the first place.

**Comment by 4dplanner** [comment]
Space lube is already in plants as well ;)

**Comment by Tlaltecuhtli** [comment]
blackpowder plants?

**Comment by Angustmeta** [comment]
> As for thermite, read the post - anyone with a chem dispenser and and some beakers can cause much more damage, much faster. 20u isn't enough to destroy a wall as of the last time I used it.

5u thermite melts a wall. 
20u thermite melts a reinforced wall.

**Comment by BebeYoshi** [comment]
Are you sure you want to add oxygen to plants? :^)

File: code/modules/hydroponics/grown/flowers.dm [hunk 1]
```diff
@@ -141,7 +141,7 @@
 	icon_dead = "sunflower-dead"
 	product = /obj/item/reagent_containers/food/snacks/grown/moonflower
 	mutatelist = list()
-	reagents_add = list("moonshine" = 0.2, "vitamin" = 0.02, "nutriment" = 0.02)
+	reagents_add = list("salbutamol" = 0.05, "oxygen" = 0.1, "nicotine" = 0.08, "nutriment" = 0.03)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/hydroponics/grown/flowers.dm]: you lost moonshine and added nicotine

Comment (ExcessiveUseOfCobblestone) [reply code/modules/hydroponics/grown/flowers.dm]: replace salbu with perf, see #42528

File: code/modules/hydroponics/grown/tobacco.dm [hunk 2]
```diff
@@ -32,7 +32,7 @@
 	plantname = "Space Tobacco Plant"
 	product = /obj/item/reagent_containers/food/snacks/grown/tobacco/space
 	mutatelist = list()
-	reagents_add = list("salbutamol" = 0.05, "nicotine" = 0.08, "nutriment" = 0.03)
+	reagents_add = list("salbutamol" = 0.05, "oxygen" = 0.1, "nicotine" = 0.08, "nutriment" = 0.03)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/hydroponics/grown/tobacco.dm]: same here regarding salbu

File: code/modules/hydroponics/grown/flowers.dm (hunk 1 above)
Comment (WaterStBlues) [reply code/modules/hydroponics/grown/flowers.dm]: Ooh shit how sloppy of me, good catch.

**Comment by dr-bee** [comment]
> 
> 
> I agree with @wesoda25 , I think this is a step in the wrong direction for botany.
> Botany is becoming a grow-your-own chemistry set, just with different ways of administering chemicals.

Isnt that the direction oranges wanted to go with chemistry? Remove the dispenser and add plants and xenobio as the primary source of chemicals.

Unless I misread the forum thread.

**Comment by uBadush** [comment]
you cannot add steel caps to grinder

**Comment by uBadush** [comment]
now we can make sorium, soap, meat, stabilizing agent(we cant heat plant chemicals though), thermite, miner salve(need a patch though), gold solidification( i think this is the best one), chloral hydrate(really small amounts like 3 units  per plant, not worth), saltpetre, Cryptobiolin, impedrezene(need omega weed but omega weed already offer options to cause massive brain damage)

**Comment by 4dplanner** [comment]
Steel cap -> hatchet -> welder -> grind

**Comment by BebeYoshi** [comment]
This may need some testing, but I'm pretty sure that if you put ALL the brain damaging chems possible on plants it's a certain death without any warnings at all, only countered by hardsuits. (As of now they are: Meth, Bath Salts, Mercury and Earthsblood which combined has a random chance to kill in one throw/slip, with the update it's also possible to mix Impedrezene, which may make it possible to kill in one throw/slip everytime)

**Comment by WaterStBlues** [comment]
> This may need some testing, but I'm pretty sure that if you put ALL the brain damaging chems possible on plants it's a certain death without any warnings at all, only countered by hardsuits. (As of now they are: Meth, Bath Salts, Mercury and Earthsblood which combined has a random chance to kill in one throw/slip, with the update it's also possible to mix Impedrezene, which may make it possible to kill in one throw/slip everytime)

new meta

**Comment by uBadush** [comment]
Meth give ultra speed bath salts make you stun immune if you die after someone inject you with it you are weak as fuck, you could run to medbay really fast

**Comment by ExcessiveUseOfCobblestone** [comment]
> can't make steel cap into grinder

who cares you just need to make a seed to transfer the iron gene to a less trash plant anywho

**Comment by BebeYoshi** [comment]
@uBadush there is no easy way to fix brain damage, at least not enough to prevent you from dying, good luck with your plan though

**Comment by Shadowflame909** [comment]
Botany replaces chemistry entirely and becomes a new involved chemistry when

**Comment by uBadush** [comment]
I don't like the idea of botany replacing chemistry entirely

**Comment by WaterStBlues** [comment]
This should be merge-ready once Travis stops being a dick

**Comment by uBadush** [comment]
What about adding plasma related plants that grow on trays that replaced the water with liqud plasma?

**Comment by WaterStBlues** [comment]
I've thought about that uBadush, considered allowing botany to grow resources much like the VGP resource mod in Rimworld (plasteel shards, uranium bulbs, etc.) Right now though, botany is currently a hot topic in terms of where Oranges and the rest of the codestapo see its place in the future of the game. It's already looking toward more unique chem production, putting it more in a xenobio niche than traditional hydroponics.

I think this could be possible, but it'd have to be very carefully balanced so that botanists and one engineer can't make mining and chemistry obsolete at the same time.

Nonetheless, the tray types would have to be something produced within botany that would take a major investment of resources and time before paying off. It'd be cool if botany had more of a tech tree system, maybe tied to science where they could specialize into resource production vs. chem production vs. production of cool niche shit like xenobio does.

For total resource production in hydro to fit neatly into the game without creating issues, a lot of overhaul would be needed.

A cool application for this tech tree idea is that botany wouldn't just become a part of science, they could choose which departments to align goals with (engi/cargo with resources, med/general population with chems and security/science with the xenobio-esque route.)
Botany always tends to shine late game if at all, so allowing them to build towards a broader goal/impact on the round would be pretty neat.

**Comment by uBadush** [comment]
I was not thinking about botany making ores but just a new branch with new plants and traits as new chemical traits, I find a bad idea to give botany power to produce resources like plasteel, uranium etc, xenobio and miners can already do that, there is nothing new about that

**Comment by optimumtact** [comment]
I don't see how I can merge this without also reworking chemistry and botany to be closer together.

**Comment by uBadush** [comment]
Tbh I don't see why would you need to rework chemistry and botany to be closer to merge this, this PR just give functions to non used plants and none of them are game breaking just simple stuff

**Comment by optimumtact** [comment]
Because it's stealing job content from another job

**Comment by uBadush** [comment]
Makes sense and I kinda agree, but this change won't be impactful enough to compete with chemistry power, I would like to see this merged because I would have more options  of stuff to do 

You can get food with slimes same as botany, you can get metal, plastel, plasma  with slimes same as miner, writing this I just realized xenobio can fill almost any function

**Comment by Shardj** [comment]
> Makes sense and I kinda agree, but this change won't be impactful enough to compete with chemistry power, I would like to see this merged because I would have more options of stuff to do
> 
> You can get food with slimes same as botany, you can get metal, plastel, plasma with slimes same as miner, writing this I just realized xenobio can fill almost any function

Xenobio can get some of the materials that miners can, but not even close to all of them, or do any of the other mining shenanigans. And getting regular food isn't what botany or xenobio does is it, these are poor comparisons.

**Comment by ExcessiveUseOfCobblestone** [comment]
Botany is already a better chem anyways in most aspects :P

**Comment by ExcessiveUseOfCobblestone** [comment]
I'll reopen once an established bot/chem relationship is made so it makes sense to give botany all sorts of chems.

**Comment by Bawhoppen** [comment]
@shardj I am pretty sure that Xenobio can get all minerals besides titanium and bananium through gold slime, but I have not played it for awhile.

**Pull request closed without merging**